website/docs/installation/configuration.mdx

```diff
@@ -77,6 +77,20 @@ To check if your config has been applied correctly, you can run the following co
 -   `AUTHENTIK_POSTGRESQL__SSLCERT`: Path to x509 client certificate to authenticate to server
 -   `AUTHENTIK_POSTGRESQL__SSLKEY`: Path to private key of `SSLCERT` certificate
 
+Additionally, databases used only for read operations can be configured. Increase the number in the following configuration variables for each read replica.
+
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__HOST`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__NAME`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__USER`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__PORT`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__PASSWORD`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__SSLMODE`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__SSLROOTCERT`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__SSLCERT`: same as above
+-   `AUTHENTIK_POSTGRESQL__READ_REPLICAS__0__SSLKEY`: same as above
+
+Note that `USE_PGBOUNCER` and `USE_PGPOOL` are inherited from the main database configuration and are not overridable per read replica. By default, if read replicas are configured, the main database is not used for reads. If you'd like it to be included for reads, add it as a read replica.
```

Comment:
I'm not sure on this; I think it both makes sense to use the primary for reading but I also think it makes sense to exclude it

Aside from that, we should also explicitly state that you can rotate the postgres credentials and authentik will pick it up, however we should also note that you cant add/remove replicas without a restart

Reply:
> I'm not sure on this; I think it both makes sense to use the primary for reading but I also think it makes sense to exclude it

Yeah, I defaulted to not include it because that's how it was written initially. I think it's fine as-is, and we can let the user make the call of whether they want to include it or not.

> Aside from that, we should also explicitly state that you can rotate the postgres credentials and authentik will pick it up, however we should also note that you cant add/remove replicas without a restart

Will do.